src/main/java/duke/exceptions/DukeWriteFailException.java

```diff
@@ -0,0 +1,10 @@
+package duke.exceptions;
+
+/**
+ * Exception to handle event where modified data cannot be written back into data file.
+ */
+public class DukeWriteFailException extends DukeException {
+    public DukeWriteFailException() {
+        super.message = "For some reason, I was unable to write the changes back to the data file!! :O";
+    }
+}
```

Comment:
Great considerations for all exception cases under DukeException!!

Reply:
Thank you hung nham.